addons/base_import_module/models/ir_module.py

```diff
@@ -141,14 +141,17 @@ def _import_module(self, module, path, force=False, with_demo=False):
                     elif ext == '.xml':
                         convert_xml_import(self.env, module, fp, idref, mode, noupdate)
                         if filename in exclude_list:
-                            self.env['ir.model.data'].create([{
-                                'name': f"{module}_{key}",
-                                'model': self.env['ir.model.data']._xmlid_lookup(f"{module}.{key}")[0],
-                                'module': "__cloc_exclude__",
-                                'res_id': value,
-                            } for key, value in idref.items() if not self.env.ref(
-                                f"__cloc_exclude__.{module}_{key}", raise_if_not_found=False
-                            )])
+                            for key, value in idref.items():
+                                name = f"{module}.{key}" if '.' not in key else key
+                                name_ = name.replace('.', '_')
+                                if self.env.ref(f"__cloc_exclude__.{name_}", raise_if_not_found=False):
+                                    continue
+                                self.env['ir.model.data'].create([{
+                                    'name': name_,
+                                    'model': self.env['ir.model.data']._xmlid_lookup(name)[0],
+                                    'module': "__cloc_exclude__",
+                                    'res_id': value,
```

Comment:
```suggestion
                                xml_id = f"{module}.{key}" if '.' not in key else key
                                name = xml_id.replace('.', '_')
                                if self.env.ref(f"__cloc_exclude__.{name}", raise_if_not_found=False):
                                    continue
                                self.env['ir.model.data'].create([{
                                    'name': name,
                                    'model': self.env['ir.model.data']._xmlid_lookup(xml_id)[0],
                                    'module': "__cloc_exclude__",
                                    'res_id': value,
```

Nitpicking but name and name_ is not very easy to read

Reply:
Indeed, thanks!